python_files/generate_supp_figure_1.py

```diff
@@ -37,9 +38,14 @@
 metadata = metadata.loc[metadata.MIBI_data_generated, :]
 metadata = metadata.loc[metadata.Timepoint.isin(['primary', 'baseline', 'pre_nivo', 'on_nivo']), :]
 
+# FOV counts per patient and timepoint
+clinical_data = pd.read_csv(os.path.join(BASE_DIR, 'intermediate_files/metadata/patient_clinical_data.csv'))
```

Comment:
This file structure is what users will see if they download the zip file from zenodo, right? Or is there any other modification/tweaking that needs to happen for these scripts to run

Reply:
A good amount of the supplementary figures need additional data files generated and located on the NAS, but not included in the zenodo upload. But yes, you should be all set to run with the current paths.

Reply:
These files are generated as part of the code that will be in the publications repo?

Reply:
The code for those is in generate_review_figures.py right now, but I can go through and tighten up the code in a separate PR. For generating SpaceCat features & predictions with different data sets (dropping low cellularity images, dropping features, etc.) it requires running SpaceCat, processing scripts 6 and 7, along with the R scripts. We discussed before putting the code from 6/7 into easy to use functions so I can definitely do that.